dev-libs/librelp/librelp-1.2.10.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit autotools
+
+DESCRIPTION="An easy to use library for the RELP protocol"
+HOMEPAGE="http://www.librelp.com/"
+SRC_URI="http://download.rsyslog.com/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-3 doc? ( FDL-1.3 )"
```

Comment:
Thanks! I pushed another commit which will adjust the license for each librelp ebuild in tree (I verified that GPL-3+ also applies to 1.2.7).